### ISSUE TITLE
Org pages - minor design changes

### ISSUE THREAD

**Issue by mariongalley** [ThreeSixtyGiving/grantnav#878]
### To do
- [ ] Org page "Type" Link styling - make links more obvious
  - I'm not sure if it's an issue we can fix at design system level, and I'm getting deja vu from links being insufficiently obvious in previous tools developments (maybe the standard docs site for publisher guidance)
  - Links aren't clearly visible, need to either change the colour to contrast more or make them underlined - also is there a way to make it clear they take you to anchors on the page?
- [ ] Org page "Additional Data" - make context/sources more prominent
    - On the Grant page, there's a teal info style box that says where the data comes from *above* the additional data itself. On the Org page, this information is right at the bottom of the data and is relatively plain text. Could this be styled in the same way?
- [ ] Org page Funder/Recipient data wrapping - remove data and change wrapping logic
    - Layout of values on the Funder and Recipient sections wrap in a way that makes it harder to understand the figures at a glance, especially when the funder gives in multiple currencies when it is hard to see which figures relate to each other.
    - Remove **Average Award** which is a controversial stat with funders which we have already been asked to remove from 360Insights and **Greatest Award** as there isn’t a smallest award anymore - to free up space for better layout
    - Force information broken by currency to sit on the same line, or force line breaks between currencies.

### Images
  - Org page link styling
      -  ![image](https://user-images.githubusercontent.com/82958988/197172719-4a6439bb-399e-4ea6-a937-c7cf082a2367.png)
  - Additional data info:
     - Org page: 
    ![image](https://user-images.githubusercontent.com/82958988/197172867-36ed07e9-06e5-430e-b83a-d9bfb47dbd90.png)
     - Grant record:
    ![image](https://user-images.githubusercontent.com/82958988/197173122-79755f31-66ed-4e90-b7e9-9876e6022d4c.png)
 - Org page wrapping
    - As Funder ![image](https://user-images.githubusercontent.com/82958988/197174379-5cf2dd70-ccc3-471d-b913-b70ddd91ecc5.png) 
    - As Recipient ![image](https://user-images.githubusercontent.com/82958988/197174539-3f7d6407-0ede-4175-a7b7-e427b5a5fd63.png)

**Comment by michaelwood** [comment]
(1) related: https://github.com/ThreeSixtyGiving/360-ds/issues/59

(3) We currently don't have an obvious widget to replace this with and could do with some generic row grids

**Comment by mariongalley** [comment]
> (3) Not able to do currently - We currently don't have an obvious widget in the design system to replace this with and could do with looking at some generic row or grid layout

@michaelwood I guess it depends on how complex that task is whether we think it's worth pursuing - it's fairly important as it's not purely an aesthetic consideration but makes the data genuinely confusing to look at.

**Comment by michaelwood** [comment]
I haven't had any ideas on how to redesign the layout, we could try a table or something less responsive if currencies > 1 . 

With removing the stats, we do have the min award amount if we want that to be added in place of average? (full disclosure I might be slightly clinging on to those, as it took about 2 days to figure out how pre-generate them in the datastore at scale)

**Comment by KDuerden** [comment]
> With removing the stats, we do have the min award amount if we want that to be added in place of average? (full disclosure I might be slightly clinging on to those, as it took about 2 days to figure out how pre-generate them in the datastore at scale)

I support retaining greatest stat if adding min award is a possibility, and especially if that helps with balancing the design.

> try a table or something less responsive if currencies > 1 .
For clarity - is a table something that would be responsive?

For As Funder view - single currency funders, there is already a two lines of four stats.
![image](https://user-images.githubusercontent.com/20462640/200818647-84ec8993-bb93-41cd-8be6-4c871c8c5e77.png)

If we are happy to substitute average for smallest (and I would suggest order smallest, largest (to mirror earliest and latest date), then is the issue that we need to be able to force funders with > 1 currency to also have rows with 4 stats.

**Comment by KDuerden** [comment]
For **As Recipient** the same 4 stats could appear as for Funders in the currency breakdown (number in currency, amount in currency, smallest, largest). 

For the top row, could the unique Funders count be inserted next to Total, and so mirror Funder top row?

**Comment by mariongalley** [comment]
@michaelwood Is that the information you need? Are you able/happy to make a call about how to implement ~ something ~ that means information about different currencies doesn't appear in the same row as information about other currencies?

**Comment by michaelwood** [comment]
> @michaelwood Is that the information you need? Are you able/happy to make a call about how to implement ~ something ~ that means information about different currencies doesn't appear in the same row as information about other currencies?

Yes I think so. thanks.

**Comment by michaelwood** [comment]
Funder with > 1 currencies 
![image](https://user-images.githubusercontent.com/493697/201166038-7a5a5c50-b955-48d2-a46a-526df28f3df9.png)

* Added some copy " ARCADIA makes grants in 3 currencies."  to help break up the transition between display types.

Funder with just one currency 
![image](https://user-images.githubusercontent.com/493697/201168117-241b6706-3cda-4c6d-a158-f1274491ff30.png)
* Put all the stats into the same grid
* Removed the total grants in this currency box (as if just one currency total grants in currency = total grants )

The default table from the design system is actually more responsive than I knew about so that's good.

e.g. 

![image](https://user-images.githubusercontent.com/493697/201171086-77fbd063-eb9a-4927-9618-97959bfc4ba4.png)

**Comment by michaelwood** [comment]
If you're happy with the above design I will go ahead and implement it properly ^

**Comment by KDuerden** [comment]
I think putting the currency breakdown in that table looks good, and definitely improves the readability.

I would prefer the rows to default to 4 values per row because it would keep the same stats on the same rows, but it's not essential because the rest of the design is much clearer.

Just to confirm the recipient counts are 0 because this is a draft design, and will be showing in the proper implementation?

**Comment by michaelwood** [comment]
Yes the recipient is 0 here as my local test grantnav doesn't have all the data. This change is now deployed on the dev server for both funders and recipients.

Regarding the 4 values per row thing, guessing this is the stats under the "Summary" heading. The layout is controlled by the design system's 'media-card box container', which determines that boxes wrap if the each box's width cannot be maintained at a width of 33%. Like everything it's possible to change it, it would require an audit of where the widget is in use in other places as well as developing the new rules for the 'media-card box container'.

**Comment by michaelwood** [comment]
4/4 now done and on dev site
